docs/content/releases.md

```diff
@@ -7,6 +7,36 @@ toc: true
 docs: "DOCS-616"
 ---
 
+## NGINX Ingress Controller 2.2.0
+
+12 Apr 2022
+
+FEATURES:
+* [2576](https://github.com/nginxinc/kubernetes-ingress/pull/2576) Add support for IPv6.
+* [2572](https://github.com/nginxinc/kubernetes-ingress/pull/2572) Automate provisioning of Certificate resources for VirtualServer resources using cert-manager.
+
+ENHANCEMENTS:
```

Comment:
Is it just me or does this section feel light?
This all appears to be recent work, I could have sworn there was more.